scripts/release-redhat

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -euo pipefail
+set -x
+
+# Updates the redhat-cert branch on github with the newer version of the agent
```

Comment:
Protect the branch in github from deletion if you haven't already.

Reply:
It will do a reset/creation of it (the `-B` flag to `git checkout`) when this script is run so it can be deleted at will as long as it doesn't happen before the build starts in Redhat's servers.